### ISSUE TITLE
make manager option to draggable and droppable optional

### ISSUE THREAD

**Issue by FlorianLudwig** [GreyRook/PIXI.draggable#2]
This is really ugly with the current API

**Comment by FabianElsmer** [comment]
Yeah, I guess you can just replace all the this.DragAndDropManager with PIXI.DragAndDropManager, not sure why I did it this way anymore or if there ever was a reason, but it should not make a difference

**Comment by FabianElsmer** [comment]
The reason for this ugly hack seems to partly be the InteractionManager, in PIXI.draggable v2 we accessed the hitTest function of the interactionManager in functions like onDrag, now when porting to v3 I could have probably accessed it in a static way too (not sure why I didn't do that). However nowadays the interactionManager doesn't even have a hitTest function anymore, so we should get that from somewhere else.

Also we still use the refresh function of the DragAndDropManager, to get all the draggable and droppable items without having to hack into the interactionManagers rebuildInteractiveGraph.
